assets/custom.css

```
@@ -2,67 +2,69 @@ html {
   position: relative;
   min-height: 100%;
   min-width: 100%;
+  margin: 0;
 }
 body {
-  /* Margin bottom by footer height */
   margin-bottom: 60px;
   background: rgb(63,26,94);
   background: linear-gradient(180deg, rgba(63,26,94,1) 50%, rgba(38,8,43,1) 100%);
 }
-
 :root {
   --jumbotron-padding-y: 3rem;
 }
-
 .jumbotron {
   padding-top: var(--jumbotron-padding-y);
   padding-bottom: var(--jumbotron-padding-y);
   margin-bottom: 0;
   background-color: #fff;
-  background-color: #1c0a26 !important;
-  width: 75% !important;
+  background-color: #1c0a26;
+  width: 70%;
 }
-@media (min-width: 768px) {
-  .jumbotron {
-    padding-top: calc(var(--jumbotron-padding-y) * 2);
-    padding-bottom: calc(var(--jumbotron-padding-y) * 2);
-  }
-}
-
 .jumbotron p:last-child {
   margin-bottom: 0;
 }
-
 .jumbotron-heading {
   font-weight: 300;
```

Tambahin margin biar ga mepet sama logo

```suggestion
margin-top: 20;
font-weight: 300;
```